examples/dp_fed_examples/instance_level_dp/server.py

```diff
@@ -61,43 +56,6 @@ def fit_config(
     )
 
 
-class CifarInstanceLevelDPServerWithCheckpointing(InstanceLevelDpServer):
```

Comment:
All functionality implemented here is now neatly contained in the parent class of `InstanceLevelDpServer`